src/com/company/FileUtils.java

```diff
@@ -2,67 +2,49 @@
 
 import java.io.File;
 import java.io.FileWriter;
-import java.io.IOException;
+import java.util.ArrayList;
 import java.util.List;
 import java.util.Scanner;
-import static com.company.Main.*;
-
 
 public class FileUtils {
-    // method to give name for file where sorted numbers will be written(if that file name is empty)
-    public static boolean isFileExists(){
-
-        boolean fileExist = false;
 
-        try{
-            String d = fileToSave.toString();
-            if(d.equals(".txt")) {
-                fileToSave = new File("./results/sortedNumbers.txt");//
-            }
-            fileExist = true;
+    public static List<Integer> unsortedNumbers;
 
-        }
-        catch (Exception e){
-            System.out.println(e.getMessage());
-        }
-        return fileExist;
-    }
     //method to get unsorted list of numbers, reading numbers from file-source
-    public static List<Integer> getUnsortedNumbers(File fileToRead){
+    public static List<Integer> getUnsortedNumbers(File fileToRead) {
 
-        //unsortedNumbers = new ArrayList<>();
-        try{
-            Scanner scannerForNumbers = new Scanner(fileToRead);
-            int i = 0 ;
-            while (scannerForNumbers.hasNextInt()){
-                unsortedNumbers.add(i++, scannerForNumbers.nextInt());
-            }
-        }
+        unsortedNumbers = new ArrayList<>();
 
-        catch (NumberFormatException t){
+        try {
+            Main.scanner = new Scanner(fileToRead);
```

Comment:
File utils doesn't know about Main class. scanner= new scanner

Reply:
Fixed. Class Main  knows classes FileUtils and Sortings. Classes FileUtils and Sortings doesn't knows any classes.